Extensions/SystemInfo/systeminfotools.js

```diff
@@ -36,4 +36,12 @@ gdjs.evtTools.systemInfo.isMobile = function() {
  */
 gdjs.evtTools.systemInfo.isWebGLSupported = function(runtimeScene) {
 	return runtimeScene.getGame().getRenderer().isWebGLSupported();
-};
\ No newline at end of file
+};
+
+/**
+ * Checks gdjs.isPreview.
```

Comment:
Don't comment [what or how the code](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/) is doing - people can just read the code if they want to know.
Instead, comment about the why or the result: "Check if the game is run as a preview in an editor".